index.html

```diff
@@ -1,171 +1,435 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+
 <head>
-<meta charset="UTF-8">
-<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="generator" content="Asciidoctor 1.5.2">
-<title>Kill Bill userguide</title>
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400">
-<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
-<link rel="stylesheet" href="https://cdn.staticaly.com/gh/afeld/bootstrap-toc/v1.0.0/dist/bootstrap-toc.min.css">
-<link rel="stylesheet" href="/stylesheets/asciinema-player.css">
-<link rel="stylesheet" href="/stylesheets/kb.css">
+  <meta charset="UTF-8">
+  <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="generator" content="Asciidoctor 1.5.2">
+  <title>Kill Bill userguide</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css"
```

Comment:
Do you think we should already upgrade to `v5.0.0-beta1` (or at least upgrade to `v4.5.3`)? Since we're about to make CSS changes, it might be worth using a more recent version?

This could also be one in another PR / in the future. I just wanted to mention that we're running an old version today.

Reply:
@pierre Are you talking about bootstrap version?

Reply:
Yes: `4.1.1` is quite old already.

Reply:
Yes! It supports the latest bootstrap version.

Reply:
I have currently, used bootstrap version: 5.0.0-alpha2